plugins/Files/js/browser.js

```diff
@@ -375,7 +376,7 @@ var browser = {
 			// Place siafile to location
 			tools.notify(`Sharing ${label} to ${destination}`, 'siafile');
 			loader.shareDotSia(paths, destination, function() {
-				tools.notify(`Put ${label}'s .sia files at ${destination}`, 'success');
+				tools.notify(`Put ${label}s .sia files at ${destination}`, 'success');
```

Comment:
is it correct to be removing the apostrophy? What sorts of strings does ${label} expand to?